lib/core/src/lib/card-view/components/card-view-dateitem/card-view-dateitem.component.html

```diff
@@ -1,129 +1,151 @@
-    *ngIf="showProperty && !isEditable"
-    [attr.for]="'card-view-dateitem-' + property.key"
-    [ngClass]="{ 'adf-property-readonly-value': isReadonlyProperty, 'adf-property-value-editable': editable }"
-    [title]="'CORE.METADATA.ACTIONS.COPY_TO_CLIPBOARD' | translate"
->
-    {{ property.label | translate }}
-</mat-label>
 
-<div class="adf-property-value" [ngClass]="{ 'adf-property-value-editable': editable, 'adf-property-readonly-value': isReadonlyProperty }">
-    <span *ngIf="!isEditable && !property.multivalued" [attr.data-automation-id]="'card-' + property.type + '-value-' + property.key">
+<mat-form-field
+    *ngIf="showProperty && !isEditable && !property.multivalued"
+    [attr.data-automation-id]="'card-' + property.type + '-value-' + property.key"
+    (dblclick)="copyToClipboard(property.displayValue)"
+    [ngClass]="{ 'adf-property-read-only': !isEditable }"
+    class="adf-property-field">
+    <mat-label
+        class="adf-property-label"
+        [ngClass]="{
+                    'adf-property-value-editable': editable,
+                    'adf-property-readonly-value': isReadonlyProperty
+                }"
+        [attr.data-automation-id]="'card-dateitem-label-' + property.key"
+        [attr.for]="'card-view-dateitem-' + property.key"
+        [title]="'CORE.METADATA.ACTIONS.COPY_TO_CLIPBOARD' | translate"
+    >
+        {{ property.label | translate }}
+    </mat-label>
+    <input
+        matInput
+        [value]="property.displayValue"
+        title="{{ property.label | translate }}"
+        [attr.id]="'card-view-dateitem-' + property.key"
+        [attr.tabIndex]="-1"
+        [attr.aria-label]="property.label | translate"
+        class="adf-property-value"
+        [ngClass]="{
+                'adf-property-value-editable': editable,
+                'adf-property-readonly-value': isReadonlyProperty,
+            }"
+        readonly
+        disabled
+    />
+</mat-form-field>
+
+<div
+    *ngIf="!showProperty && !property.multivalued"
+    class="adf-property-value"
+    [ngClass]="{ 'adf-property-value-editable': editable, 'adf-property-readonly-value': isReadonlyProperty }"
+    >
+    <span [attr.data-automation-id]="'card-' + property.type + '-value-' + property.key"></span>
+</div>
+
+<mat-form-field *ngIf="isEditable && !property.multivalued" class="adf-dateitem-editable adf-property-field hxp-input">
+    <mat-label
+        class="adf-property-label"
+        [attr.data-automation-id]="'card-dateitem-label-' + property.key"
+        [attr.for]="'card-view-dateitem-' + property.key"
+        [ngClass]="{ 'adf-property-readonly-value': isReadonlyProperty, 'adf-property-value-editable': editable }"
+        [title]="'CORE.METADATA.ACTIONS.COPY_TO_CLIPBOARD' | translate"
+    >
+        {{ property.label | translate }}
+    </mat-label>
+    <div class="adf-dateitem-editable-controls">
         <span
-            *ngIf="showProperty"
```

Comment:
as above, try to use control flow syntax so that you can get rid of common module dependency